src/java/us/kbase/templates/module_dockerfile.vm.properties

```diff
@@ -2,13 +2,24 @@ FROM kbase/kbase:sdkbase.latest
 MAINTAINER #if($username)${username}#{else}KBase Developer#{end}
 
 # -----------------------------------------
-
-# Insert apt-get instructions here to install
-# any required dependencies for your module.
+# In this section, you can install any system dependencies required
+# to run your App.  For instance, you could place an apt-get update or
+# install line here, a git checkout to download code, or run any other
+# installation scripts.  Here we install a python coverage tool and an
+# https library that is out of date in the base image.
 
 # RUN apt-get update
 #if( $language == "python")
+
 RUN pip install coverage
+
+# update security libraries in the base image
+RUN pip install cffi --upgrade \
+    && pip install pyopenssl --upgrade \
+    && pip install ndg-httpsclient --upgrade \
```

Comment:
yes- we ran into the same problem and this is a fix courtesy of Gavin.  @MrCreosote you actually don't need the first sudo apt-get install line because those things are already up to date in the base image.